### DISCUSSION TITLE
Using multiple Firebase projects

### DISCUSSION THREAD

**Discussion by MattWellsBlue42** [capawesome-team/capacitor-firebase#668] [Q&A] [answered]
Hey everyone!

I just want to start by saying that these plugins have made integrating Firebase into my React Ionic project so much easier!

I’ve run into a challenge where we’re using two separate Firebase projects: one for authentication and user information (let’s call it Firebase 1), and another for app-specific data (let’s call it Firebase 2).

My question is: is there a way to manage this setup? I know that setting up these plugins requires using the GoogleService-Info.plist and google-services.json files. Is it possible to use two different sets of these files? Specifically, can we configure the authentication plugin to use Firebase 1’s files while configuring the Firestore plugin to use Firebase 2’s files?

Thanks in advance for your help!

**Comment by robingenz** [comment, marked as answer]
No, this is not yet possible. Feel free to create a feature request.

**Reply by MattWellsBlue42** [reply]
Thank you for the quick response!
